perspective.el

```diff
@@ -293,9 +263,9 @@ for the perspective."
       (dotimes (_ 2) (push (pop args) keywords)))
     (setq keywords (reverse keywords))
     `(let ((persp (make-persp-internal ,@keywords)))
-       (when persp-curr
-         (setf (persp-local-variables persp) (persp-local-variables persp-curr)))
-       (puthash (persp-name persp) persp perspectives-hash)
+       (when (frame-parameter nil 'persp-curr)
```

Comment:
Wrap `(frame-parameter nil 'persp-curr)` in a function so you don't have to repeat it all over the place. Same with other frequently-accessed frame parameters.

Reply:
I've added wrapper functions for getting `persp-curr` and `persp-last`.